addons/medical_gui/initSettings.sqf

```diff
@@ -126,3 +126,21 @@ private _categoryColors = [ELSTRING(medical,Category), format ["| %1 |", LELSTRI
     true,
     true // isGlobal
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(peekMedicalOnHit),
+    "CHECKBOX",
+    [LSTRING(PeekMedicalOnHit_DisplayName), LSTRING(PeekMedicalOnHit_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    true,
+    false // isGlobal
+] call CBA_fnc_addSetting;
```

Comment:
This should not be local IMO.

Some might prefer defaulting to off.

Reply:
I set it local to allow servers to leave it to player preference. Particularly thinking about if somone wants to disable it while a server allows it. But I don't feel too strongly about it.

I do agree with defaulting to off, on second thought.

Reply:
Local can always be overwritten.